app/containers/CreateWallet/OptionButton.jsx

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import React from 'react'
+import styles from '../Home/Home.scss'
+
+const OptionButton = ({ active, children, handleClick }: Object) => {
+  return (
+    <div
+      onClick={handleClick}
+      className={active ? styles.activeOptionButton : styles.optionButton}
```

Comment:
We should create a separate `OptionButton.scss` file that these styles go into.

Reply:
👍

Reply:
☝️ this has been resolved with https://github.com/CityOfZion/neon-wallet/pull/1070/commits/bef1f8aacf6aa62dd016a1c77fcff876385caff9